code/modules/mob/living/simple_animal/parrot.dm

```diff
@@ -876,11 +891,21 @@
 
 /mob/living/simple_animal/parrot/Poly/proc/Read_Memory()
 	var/savefile/S = new /savefile("data/npc_saves/Poly.sav")

```

Comment:
I tried but there's some weird hang ups with trying to define/const a savefile and/or I'm incompetent.

Considering these are both once per round events I don't think it's a big deal.